compiler/src/org.graalvm.compiler.lir/src/org/graalvm/compiler/lir/constopt/ConstantTree.java

```diff
@@ -76,7 +76,7 @@ public void addUsage(UseEntry usage) {
 
         public List<UseEntry> getUsages() {
             if (usages == null) {
-                Collections.emptyList();
+                return Collections.emptyList();
```

Comment:
@davleopo , looking at the callers of `getUsages()`, I'm very surprised this hasn't caused NPEs in the past - any idea why?